### PR TITLE
The addition to #513

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
## Change list

Addition to #513. ```IOSSwipeGestureTest ``` is updated.
 
## Types of changes

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

@TikhomirovSergey FYI

### CODE REVIEW

**Comment by TikhomirovSergey** [comment]
@SrinivasanTarget 
Everything is ok. 
Could you please add a test like that 
https://gist.github.com/TikhomirovSergey/39df3c0b67e08e3447d200cc55d26d05

for iOS?

**Comment by SrinivasanTarget** [comment]
Sure @TikhomirovSergey

**Comment by SrinivasanTarget** [comment]
@TikhomirovSergey Will complete this tomorrow. Sorry for the delay.

**Comment by SrinivasanTarget** [comment]
git checkout -b SrinivasanTarget-TikhomirovSergey-SrinivasanTarget-TouchActionSplitting master
git pull https://github.com/SrinivasanTarget/java-client.git TikhomirovSergey-SrinivasanTarget-TouchActionSplitting

File: src/main/java/io/appium/java_client/ios/IOSTouchAction.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+public class IOSTouchAction extends TouchAction implements CreatesSwipeAction {
+
+    public IOSTouchAction(PerformsTouchActions performsTouchActions) {
+        super(performsTouchActions);
+    }
+
+    @Override
+    public TouchAction swipe(int startX, int startY, int endX, int endY, int duration) {
+        int xOffset = endX - startX;
+        int yOffset = endY - startX;
```
Comment (TikhomirovSergey) [inline src/main/java/io/appium/java_client/ios/IOSTouchAction.java]: There is the bug

**Comment by SrinivasanTarget** [comment]
@TikhomirovSergey Thanks. All tests passed now.

**Pull request merged**